src/core/event_registry.rs

```diff
@@ -21,6 +21,16 @@ impl EventRegistry {
         self.client.push(event_data);
     }
 
+    pub(crate) fn make_independent(&mut self, events_id: ComponentId) {
+        self.server
+            .iter_mut()
+            .find(|event| event.events_id() == events_id)
+            .unwrap_or_else(|| {
+                panic!("event with ID {events_id:?} should be previously registered");
+            })
+            .make_independent();
```

Comment:
One minor stylistic suggestion, just a personal preference :) You can apply it from the browser.
```suggestion
        let event = self
            .server
            .iter_mut()
            .find(|event| event.events_id() == events_id)
            .unwrap_or_else(|| {
                panic!("event with ID `{events_id:?}` should be previously registered");
            });
        event.make_independent();
```

Reply:
Fixed